kapitan/targets.py

```diff
@@ -169,9 +169,10 @@ def compile_targets(
 
         # if '-t' is set on compile or only a few changed, only override selected targets
         if updated_targets:
```

Comment:
is this `if` still needed if you're no longer using that variable?

Reply:
yes it's still needed to distinguish between "all targets" and "some targets" so that we copy individual paths instead of everything